Doc/whatsnew/3.11.rst

```diff
@@ -693,8 +693,8 @@ Build Changes
 
 * Build dependencies, compiler flags, and linker flags for most stdlib
   extension modules are now detected by :program:`configure`. libffi, libnsl,
-  libsqlite3, zlib, bzip2, liblzma, libcrypt, and uuid flags are detected by
-  ``pkg-config`` (when available).
+  libsqlite3, zlib, bzip2, liblzma, libcrypt, Tcl/Tk libs, and uuid flags
+  are detected by ``pkg-config`` (when available).
```

Comment:
Somewhere  in here, we should explicitly document the removal of the `--with-tcltk-includes` and `--with-tcltk-libs` configuration options. I have seen them (and the corresponding Makefile variables) in use in the wild.

Reply:
I agree. I'll add a sentence or two to the build section of What's New. I'll also create a PR for the devguide.

Reply:
Is b48d671fe1330933322f715e3aabdb234533f38a sufficient?